### PR TITLE
add favicon and minor teak

### PR DESCRIPTION
**Pull request by eteq**
This adds a favicon (from the astronomical python twitter account), and adds just a bit more info about UW/eScience as the location. @astrofrog

### CODE REVIEW

File: index_2016.html [hunk 1]
```diff
@@ -19,7 +22,8 @@
         <p>
           The second <i>Python in Astronomy</i> conference will be held during
           the week of 21-25 March 2016 (exact start/end day to be confirmed) at
-          the University of Washington. We are aiming for a similar format to
+          the the <a href="http://escience.washington.edu/"> University of Washington eScience Institute</a>
```
Comment (astrofrog) [inline index_2016.html]: @eteq - could you fix the double 'the'?

File: index_2016.html [hunk 2]
```diff
@@ -19,7 +22,8 @@
...
+           in Seattle, WA, UA. We are aiming for a similar format to
```
Comment (astrofrog) [inline index_2016.html]: UA -> USA

**Comment by eteq** [comment]
both done!

**Comment by astrofrog** [comment]
@eteq - thanks!

**Pull request merged**